### PR TITLE
Improve `root_dir` option's description

### PR DESCRIPTION
**Pull request by ybiquitous**
This change aims to make the description more clear for users.

### CODE REVIEW

**Comment by ybiquitous** [comment]
@yykamei Could you give me a brief review, please?

File: docs/getting-started/custom-configuration.md [hunk 1]
```diff
@@ -93,15 +93,17 @@ The following sections describe the available options.
...
 
 ```yaml
 linter:
   eslint:
-    root_dir: "frontend"
+    root_dir: "frontend/"
 ```
 
-Sider will run ESLint analysis in `./frontend` directory.
+In the configuration above, Sider will run ESLint at the `frontend/` directory (perhaps will load `frontend/.eslintrc.js` or `frontend/package.json` files).
```
Comment (ybiquitous) [inline docs/getting-started/custom-configuration.md:104]: [IMO] The following seems better to me:

```suggestion
In the configuration above, Sider will run ESLint at the `frontend/` directory (perhaps will load files such as `frontend/.eslintrc.js` or `frontend/package.json`).
```

Comment (yykamei) [reply docs/getting-started/custom-configuration.md:104]: Agreed :+1:

**Review by yykamei — APPROVED** [review]
LGTM

**Comment by ybiquitous** [comment]
Thanks ❤️

**Pull request merged**